classes/user/PKPUserDAO.inc.php

```diff
@@ -230,6 +230,7 @@ function getAllReviewers($contextId) {
 	/**
 	 * Given the ranges selected by the editor, produce a filtered list of reviewers
 	 * @param $contextId int
+	 * @param $name string|null Partial string match with reviewer name
```

Comment:
Looks like there are some missing parameter docs here, but not from your addition, Nate. I'll add this when I merge.